assemblies/resources/bin/check_ports

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+# -----------------------------------------------------------------------
+# check if http or https port is available to run opencast otherwise exit
+
+if [ -d "${KARAF_ETC}" ]; then
+  SETENV_ETC="${KARAF_ETC}"
+else
+  SETENV_ETC="etc"
```

Comment:
maybe you could use the `bin/inc` to load all necessary Karaf variables.

Reply:
is bin inc executed?

Reply:
It's not executed but included to load/set the Karaf variables for multiple Karaf files (of which we only use one):

```
bin/start-opencast:. "${DIRNAME}/inc"
```

Reply:
should I move the check_ports to inc?

Reply:
hm, need to check where `inc` is actually loaded but it might work to just extend the file in the same manner we do that with some other files already:

https://github.com/opencast/opencast/blob/18c4bb9c713f9f22002425f7f313101eb32f96f7/assemblies/pom.xml#L140-L146

Reply:
I'm checking if its run from `start-opencast` -  otherwise we have to gather all the variables from scratch as `inc` is run before  `setenv`